test/e2e-go/features/transactions/asset_test.go

```diff
@@ -491,6 +491,108 @@ func TestAssetInformation(t *testing.T) {
 	a.NoError(err)
 }
 
+func TestAssetGroupCreateSendDestroy(t *testing.T) {
+	t.Parallel()
+	a := require.New(t)
+
+	var fixture fixtures.RestClientFixture
+	fixture.Setup(t, filepath.Join("nettemplates", "TwoNodes50Each.json"))
+	defer fixture.Shutdown()
+
+	client0 := fixture.GetLibGoalClientForNamedNode("Primary")
+	client1 := fixture.GetLibGoalClientForNamedNode("Node")
+
+	wh0, err := client0.GetUnencryptedWalletHandle()
+	a.NoError(err)
+	wh1, err := client1.GetUnencryptedWalletHandle()
+	a.NoError(err)
+
+	client0.ListAddresses(wh0)
+	accountList, err := client0.ListAddresses(wh0)
+	a.NoError(err)
+	account0 := accountList[0]
+
+	client1.ListAddresses(wh0)
+	accountList, err = client1.ListAddresses(wh1)
+	a.NoError(err)
+	account1 := accountList[0]
+
+	txCount := uint64(0)
+	fee := uint64(1000000)
+
+	manager := account0
+	reserve := account0
+	freeze := account0
+	clawback := account0
+
+	{
+		// Create and Send in the same group
+		assetName := "test1"
+		assetUnitName := "testunit1"
+		txCreate, err := client0.MakeUnsignedAssetCreateTx(100, false, manager, reserve, freeze, clawback, assetName, assetUnitName, "foo://bar", nil)
+		a.NoError(err)
+		txCreate, err = client0.FillUnsignedTxTemplate(account0, 0, 0, fee, txCreate)
+		a.NoError(err)
+
+		txSend, err := client1.MakeUnsignedAssetSendTx(txCount+1, 0, account1, "", "")
+		a.NoError(err)
+		txSend, err = client1.FillUnsignedTxTemplate(account1, 0, 0, fee, txSend)
+		a.NoError(err)
+
+		gid, err := client0.GroupID([]transactions.Transaction{txCreate, txSend})
+
+		var stxns []transactions.SignedTxn
+		txCreate.Group = gid
+		stxn, err := client0.SignTransactionWithWallet(wh0, nil, txCreate)
+		a.NoError(err)
+		stxns = append(stxns, stxn)
+
+		txSend.Group = gid
+		stxn, err = client1.SignTransactionWithWallet(wh1, nil, txSend)
+		a.NoError(err)
+		stxns = append(stxns, stxn)
+
+		// broadcasting group should succeed
+		err = client0.BroadcastTransactionGroup(stxns)
+		a.NoError(err)
+
+		txCount += uint64(len(stxns))
```

Comment:
You need to wait until the transaction group is confirmed an included in the block.
Just broadcasting the transaction doesn't mean it will be included in the block.

Reply:
done and added more check after txns confirmation.